azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -591,9 +597,15 @@ private void finalizeFlow(final ExecutableFlowBase flow) {
     final long durationSec = (flow.getEndTime() - flow.getStartTime()) / 1000;
     switch (flow.getStatus()) {
       case FAILED_FINISHING:
-        this.logger.info("Setting flow '" + id + "' status to FAILED in "
+        final Status status = isKilled() ? Status.KILLED : Status.FAILED;
```

Comment:
is this change related with killing status or it's just a low hanging fruit of bug fix?